pki.spec

```diff
@@ -1423,6 +1431,11 @@ fi
 %{_bindir}/PrettyPrintCert
 %{_bindir}/PrettyPrintCrl
 %{_bindir}/TokenInfo
+%{_bindir}/pki-healthcheck
+%{python3_sitelib}/pkihealthcheck/
```

Comment:
I'd suggest moving the package under pki.server.* for consistency with other server packages.